rust/phoenix-sdk-core/src/sdk_client_core.rs

```diff
@@ -130,43 +141,43 @@ impl SDKClientCore {
     }
 
     /// RECOMMENDED:
-    /// Converts quote amount to quote lots. For example if the quote currency was USDC with 6 decimals and you wanted to
-    /// convert 3 USDC to quote lots you would call sdk.quote_amount_to_quote_lots(3_000_000). This would return
-    /// the number of quote lots that would be equivalent to 3 USDC.
-    pub fn quote_amount_to_quote_lots(&self, quote_amount: u64) -> u64 {
-        quote_amount / self.quote_lot_size
+    /// Converts quote atoms to quote lots. For example if the quote currency was USDC with 6 decimals and you wanted to
+    /// convert 3 USDC, or 3_000_000 USDC atoms, to quote lots you would call sdk.quote_amount_to_quote_lots(3_000_000). This would return
```

Comment:
nit: fix doc `quote_amount_to_quote_lots`

Reply:
I believe `quote_amount_to_quote_lots` is deleted and the docs now match with the latest function name `quote_units_to_quote_lots`

Reply:
The doc says `quote_amount_to_quote_lots` still?

Reply:
Ah I see thanks! Changed in the latest commit